htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/annotations/Timeout.java

```diff
@@ -0,0 +1,12 @@
+package ru.yandex.qatools.htmlelements.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.FIELD})
+public @interface Timeout {
```

Comment:
I don't have an idea how to test it, because we don't have public api for getting timeout value from AjaxElementLocator object. Please, give me some advice.